packages/react-components/svgTransform.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  process() {
+    return {
+      code: `module.exports = {};`,
+    };
+  },
+  getCacheKey() {
+    // The output is always the same.
+    return 'svgTransform';
+  },
+};
```

Comment:
What's the point of this file? Maybe a comment, or similar, would help me understand?